### ISSUE TITLE
WCAG 2.2 Violation - There are visible labels present within the interface whose content is not included in the accessible names of the controls that they label.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#6]
### Issue

There are controls with visible labels that don't include the text of those visible labels in their accessible names.

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.5.3 Label in Name](https://www.w3.org/TR/WCAG22/#label-in-name)

### User Impact

Some users - such as people using voice control software - may be unable to access, focus, or activate the controls. Screen reader users may be confused by the controls being announced by a different accessible name than their visible labels.

#### Further information:

1. 

### Instances

1. #43 
2. #44 

#### Using the Chrome developer tools:

1. Take note of the visible text of the control's label.
2. Inspect the control using Chrome's developer tools.
3. Activate the "Accessibility" tab (for the element) within the developer tools.
4. Under "Computed Properties", notice that the "Name" value does not include the visible text of the control's label (noting that _all_ of the visible text of the control's label must be included in the control's accessible name, verbatim).

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate to the control.
3.
